src/vs/workbench/browser/parts/quickopen/quickOpenController.ts

```diff
@@ -549,6 +568,33 @@ export class QuickOpenController extends WorkbenchComponent implements IQuickOpe
 		return promiseCompletedOnHide;
 	}
 
+	private handleOnCancel(isPicker: boolean): void {
+		// restore the editor part state after cancelling
+		this.historyService.block(true);
```

Comment:
@wprater shouldn't this be `block(false)`? also I am missing the all to `block(true)`.